Moodlet/app/src/main/java/com/cmput3owo1/moodlet/models/EmotionalState.java

```diff
@@ -47,16 +43,9 @@
      */
     @Override
     public String toString(){
-        return this.display_name;
+        return this.displayName;
```

Comment:
This should return the emotional state as a string, so return the name and the color

Reply:
It needs to be just the name for purposes of the dropdown and displaying in a list